proprietary/tokens/src/brand/ams/space.compact.tokens.json

```diff
@@ -1,21 +1,12 @@
 {
   "ams": {
     "space": {
-      "xs": { "value": "0.25rem" },
-      "sm": { "value": "0.5rem" },
-      "md": { "value": "1rem" },
-      "lg": { "value": "1.5rem" },
-      "xl": { "value": "2rem" },
-      "grid": {
-        "xs": { "value": "clamp(0.25rem, calc(0.390625vw - 0.015625rem), 0.625rem)" },
-        "sm": { "value": "clamp(0.5rem, calc(0.78125vw - 0.03125rem), 1.25rem)" },
-        "md": {
-          "value": "clamp(1rem, calc(1.5625vw - 0.0625rem), 2.5rem)",
-          "comment": "Grows from 16px at 1088px wide to 40px at 2624px wide."
-        },
-        "lg": { "value": "clamp(1.5rem, calc(2.34375vw - 0.09375rem), 3.75rem)" },
-        "xl": { "value": "clamp(2rem, calc(3.125vw - 0.125rem), 5rem)" }
-      }
+      "xs": { "value": "clamp(0.1875rem, 0.1719rem + 0.0781vw, 0.25rem)" },
+      "sm": { "value": "clamp(0.375rem, 0.3438rem + 0.1563vw, 0.5rem)" },
+      "md": { "value": "clamp(0.75rem, 0.6875rem + 0.3125vw, 1rem)" },
+      "lg": { "value": "clamp(1rem, 0.875rem + 0.625vw, 1.5rem)" },
+      "xl": { "value": "clamp(1.25rem, 1rem + 1.25vw, 2.25rem)" },
+      "2xl": { "value": "clamp(1.5rem, 1.125rem + 1.875vw, 3rem)" }
```

Comment:
My only issue here is that it should be `xxl`, the actual value needs to prove itself in apps and Figma. 
I know you said `2xl` because of scalability but i would stick to t-shirt sizes if you opt to use t-shirt sizes.